src/Components/FormCreation/CardComponents/SelectQuestion.jsx

```diff
@@ -113,6 +113,12 @@ function SelectQuestion({
   });
   const [selected, setSelected] = useState(selectedInit);
 
+  useEffect(() => {
+    if (initialOptions !== options) {
```

Comment:
I don't think you can do equality like this? At least it's not working for me locally (infinite loop when making a change)